### ISSUE TITLE
Check installation of pymchelper with python installed from python.org

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/pymchelper#391]
Check installation on windows, some users reported problems there:

```
For Python questions:

    my python version: 3.7 installed from python.org
    On Windows, I also use Spyder 3.3
    For installation, I used "pip install" and "pip3 install" later. It said "installation complete" but when I try to "import pymchelper" in code then it says "no such module".

But it's no big deal because I installed it on Ubuntu VM without any problems (same python version and same command) and it works.
```

**Comment by grzanka** [comment]
Is seems to work with Python 3.9:

![image](https://user-images.githubusercontent.com/7374733/114855760-969f6700-9de6-11eb-8c92-c16e74b3b9e8.png)

**Comment by grzanka** [comment]
Python 3.7 seems also to be OK:
![image](https://user-images.githubusercontent.com/7374733/114856490-74f2af80-9de7-11eb-9fd6-a425e4ce55c7.png)

**Comment by grzanka** [comment]
Seems to work now:

```
PS C:\Users\Legion\workspace\1> .\venv\Scripts\Activate.ps1
(venv) PS C:\Users\Legion\workspace\1> pip install pymchelper
Collecting pymchelper
  Obtaining dependency information for pymchelper from https://files.pythonhosted.org/packages/ba/eb/26978181ec92b04f34fbb1b16c20e48e0ef98e694f1f8bc06a2e0ed60325/pymchelper-2.7.5-py2.py3-none-any.whl.metadata
  Downloading pymchelper-2.7.5-py2.py3-none-any.whl.metadata (5.6 kB)
Collecting numpy>=1.23.3 (from pymchelper)
  Obtaining dependency information for numpy>=1.23.3 from https://files.pythonhosted.org/packages/d0/06/3d1ff6ed377cb0340baf90487a35f15f9dc1db8e0a07de2bf2c54a8e490f/numpy-2.2.0-cp311-cp311-win_amd64.whl.metadata
  Using cached numpy-2.2.0-cp311-cp311-win_amd64.whl.metadata (60 kB)
Downloading pymchelper-2.7.5-py2.py3-none-any.whl (268 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 268.9/268.9 kB 1.5 MB/s eta 0:00:00
Using cached numpy-2.2.0-cp311-cp311-win_amd64.whl (12.9 MB)
Installing collected packages: numpy, pymchelper
Successfully installed numpy-2.2.0 pymchelper-2.7.5

[notice] A new release of pip is available: 23.2.1 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
(venv) PS C:\Users\Legion\workspace\1> convertmc --version
2.7.5
(venv) PS C:\Users\Legion\workspace\1> python
Python 3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pymchelper
>>> pymchelper.__version__
'2.7.5'
>>> exit()
(venv) PS C:\Users\Legion\workspace\1> pip uninstall pymchelper
Found existing installation: pymchelper 2.7.5
Uninstalling pymchelper-2.7.5:
  Would remove:
    c:\users\legion\workspace\1\venv\lib\site-packages\pymchelper-2.7.5.dist-info\*
    c:\users\legion\workspace\1\venv\lib\site-packages\pymchelper\*
    c:\users\legion\workspace\1\venv\scripts\convertmc.exe
    c:\users\legion\workspace\1\venv\scripts\mcscripter.exe
    c:\users\legion\workspace\1\venv\scripts\plan2sobp.exe
    c:\users\legion\workspace\1\venv\scripts\runmc.exe
Proceed (Y/n)? y
  Successfully uninstalled pymchelper-2.7.5
```
